tests/unit/services/session-test.js

```diff
@@ -41,7 +41,7 @@ describe('SessionService', () => {
 
   it('forwards the "invalidationSucceeded" event from the session', (done) => {
     let triggered = false;
-    sessionService.one('invalidationSucceeded', () => triggered = true);
+    sessionService.one('invalidationSucceeded', () => { triggered = true; });
```

Comment:
because you shouldn't `return` assignments, and the arrow function is implicitly doing that.